README.adoc

```diff
@@ -562,7 +582,7 @@ Wait a few minutes, then run the following command to see the newly scaled up `s
 oc get pods
 ```
 
-Revisit the `inventory` application and you now see three elements:
+Revisit the `inventory` microservice and you will now see three instances of the `system` microservice listed in the `inventory` endpoint:
```

Comment:
avoid future tense:

Revisit the `inventory` microservice and you can now see three instances of the `system` microservice listed in the `inventory` endpoint: